packages/styles/src/elements/body.scss

```diff
@@ -1,17 +1,28 @@
 @use 'sass:map';
+@use '../variables/color';
 @use '../functions/tokens';
 @use '../tokens/elements';
+@use '../tokens/palettes';
 
 tokens.$default-map: elements.$post-body;
 
 // Resets type
 @use '../components/fonts';
 
+html {
+  // This in combination with the default background color on the body creates a nice effect when overscrolling
+  // in Safari and Chrome on a Mac because the header and the footer are yellow, the yellow surface stretches
+  // beyond the body. If this is not yellow, overscrolling reveals a white background and the header
+  // suddenly seems in the middle of the page.
+  background-color: color.$yellow;
+}
+
 body {
   font-family: tokens.get('body-font-family');
   font-size: tokens.get('body-font-size');
   font-weight: tokens.get('body-font-weight');
   line-height: tokens.get('body-line-height');
   letter-spacing: tokens.get('body-letter-spacing');
   color: tokens.get('body-color');
+  background-color: tokens.get('palettes-color-default-bg', palettes.$post-palettes);
```

Comment:
The background color of the body is already set in the palettes PR: https://github.com/swisspost/design-system/pull/3850/files#diff-8c14f2102052982c3be175ed844b74d595f43df77e4c22707011eecdd5b4ad2dR5

```suggestion
```

Reply:
In the end, I think I had you create this PR for nothing... 🙈 I thought you would need the palettes to fetch the Cargo green for the HTML element.

Reply:
There's still the background of the `<html>` element that changes, so not entirely for nothing.